### ISSUE TITLE
SH+spacebar should trigger page up on detail view

### ISSUE THREAD

**Issue by billsaysthis** [brentsimmons/NetNewsWire#1113]
Opposite of how spacebar currently works, which I think is standard for macOS apps.

Using latest stable version of NNW, macOS 10.14.6

**Comment by JeffBerman** [comment]
I agree. I'm a first-time user of NetNewsWire and RSS, and my natural inclination was that shift-spacebar would page up, similar to web browsers. Maybe I'm a poor reader, but sometimes I race through content, only to page up later to re-read a portion of text. I haven't been able to find a keyboard shortcut for this.

I don't think that shift-spacebar needs to move to the previous article once it reaches the beginning of the current one; just paging up until the beginning of the article would be tremendous.

Thanks! I'm loving NetNewsWire!
